README.md

```diff
@@ -134,6 +136,64 @@ agent.tool.file_write(path="output.txt", content="Hello, world!")
 agent.tool.editor(command="view", path="script.py")
 ```
 
+### MCP Client Integration
+
+```python
+from strands import Agent
+from strands_tools import mcp_client
+
+agent = Agent(tools=[mcp_client])
+
+# Connect to a custom MCP server via stdio
+agent.tool.mcp_client(
+    action="connect",
+    connection_id="my_tools",
+    transport="stdio",
+    command="python",
+    args=["my_mcp_server.py"]
+)
+
+# List available tools on the server
+tools = agent.tool.mcp_client(
+    action="list_tools",
+    connection_id="my_tools"
+)
+
+# Call a tool from the MCP server
+result = agent.tool.mcp_client(
+    action="call_tool",
+    connection_id="my_tools",
+    tool_name="calculate",
+    x=10,
```

Comment:
I see here the arguments are passed as mcp_client argument for calling tools which will conflict if the target server / tool have same arguments.

For example, how the argument will apply if the argument name is tool_name or action here ^^